tests/election.py

```diff
@@ -79,7 +79,11 @@ def run(args):
         # Number of nodes F to stop until network cannot make progress
         nodes_to_stop = math.ceil(len(args.nodes) / 2)
         if args.consensus == "bft":
-            nodes_to_stop = math.ceil(len(args.nodes) / 3)
+            # nodes_to_stop = math.ceil(len(args.nodes) / 3)
```

Comment:
f=1 (line 132), so this should be 1 currently anyway?

Reply:
This is actually 2. When view-changes are fully implemented we start the test with 4 nodes, the first view-change will success and the second will fail.

Reply:
Ok, we parametrise on f already, so perhaps we just pass that in to run() and use it directly here rather than back it out of the number of nodes.